GenderPayGap.WebUI/BusinessLogic/Services/UpdateFromCompaniesHouseService.cs

```diff
@@ -167,7 +167,7 @@ private static bool IsNewOrganisationAddressNullOrEmpty(OrganisationAddress addr
 
         public static OrganisationAddress CreateOrganisationAddressFromCompaniesHouseAddress(CompaniesHouseAddress companiesHouseAddress)
         {
-            string premisesAndLine1 = GetAddressLineFromPremisesAndAddressLine1(companiesHouseAddress);
+            string premisesAndLine1 = companiesHouseAddress.GetAddressLineFromPremisesAndAddressLine1();

```

Comment:
I thought it was cleaner for this function to live within the `CompaniesHouseAddress` class rather than as bloaty code in the service that can't be reused elsewhere (not that I know if this'll be useful anywhere else!)